openstack/css/v1/clusters/AddClusterNodes.go

```diff
@@ -0,0 +1,84 @@
+package clusters
+
+import (
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/internal/build"
+	"github.com/opentelekomcloud/gophertelekomcloud/internal/extract"
+)
+
+// ScaleInOpts defines options for scaling in a cluster.
+type AddNodesOpts struct {
+	// NodeSize - Number of nodes. The value range is 1 to 32.
+	// If the node type is ess-master, the number of nodes must be an odd number in the range 3 to 10.
+	// If the node type is ess-client, the number of nodes must be in the range 1 to 32.
+	NodeSize int `json:"node_size" required:"true"`
+	// Flavor - Flavor ID.
+	Flavor string `json:"-" required:"true"`
+	// Type of the volume.
+	// One of:
+	//   - `COMMON`: Common I/O
+	//   - `HIGH`: High I/O
+	//   - `ULTRAHIGH`: Ultra-high I/O
+	VolumeType string `json:"volume_type" required:"true"`
+}
+
+type apiAddNodesOpts struct {
+	// NodeSize - Number of nodes. The value range is 1 to 32.
+	// If the node type is ess-master, the number of nodes must be an odd number in the range 3 to 10.
+	// If the node type is ess-client, the number of nodes must be in the range 1 to 32.
+	NodeSize int `json:"node_size" required:"true"`
+	// ID of the new flavor.
+	FlavorRef string `json:"flavor_ref" required:"true"`
+	// Type of the volume.
+	// One of:
+	//   - `COMMON`: Common I/O
+	//   - `HIGH`: High I/O
+	//   - `ULTRAHIGH`: Ultra-high I/O
+	VolumeType string `json:"volume_type" required:"true"`
+}
+
+func AddClusterNodes(client *golangsdk.ServiceClient, clusterID string, NodeType string, opts AddNodesOpts) (*AddNodesResponse, error) {
+	var (
+		url      string
+		flavorID string
+		err      error
+		res      AddNodesResponse
+	)
+
+	flavorID, err = getFlavorIDByName(client, opts.Flavor, NodeType)
+	if err != nil {
+		return &res, err
+	}
+
+	apiOpts := apiAddNodesOpts{
+		NodeSize:   opts.NodeSize,
+		VolumeType: opts.VolumeType,
+		FlavorRef:  flavorID,
+	}
```

Comment:
remove this from here

Reply:
functions must be atomic only one exact request per func

Reply:
Updated the function to use exact request as mentioned in the document.